.github/workflows/tact.yml

```diff
@@ -40,11 +46,28 @@ jobs:
         run: |
           yarn cross-env echo $NODE_ENV
 
-      - name: Build and Test Tact compiler
+      - name: Build Tact compiler
         run: |
           yarn clean
           yarn gen
           yarn build
+
+      - name: Verify fresh compiled stdlib
+        if: runner.os != 'Windows'
+        run: |
+          FILE_PATH="src/imports/stdlib.ts"
+          ORIGINAL_CONTENT="${{ steps.capture_stdlib.outputs.original_content }}"
+          CURRENT_CONTENT=$(cat "$FILE_PATH")
+
+          if ! cmp -s "$FILE_PATH" "$FILE_PATH.original"; then
```

Comment:
you can probably just use `diff` -- it returns a non-zero exit code if two files are not equal (you won't have to store the contents of the files as separate steps)